### PR TITLE
Add the option to use ARM64

### PR DESCRIPTION
**Pull request by jvisker**

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -1,7 +1,7 @@
 terraform {
   required_version = ">= 0.12.24"
   required_providers {
-    aws = ">= 2.58"
+    aws = ">= 3.69"
```
Comment (GaryGSC) [inline main.tf:4]: This is a breaking change. I'd call this `v3.0.0`.

**Comment by jvisker** [comment]
Helps solve an issue here: https://github.com/byu-oit/appeng-questions/discussions/60

File: README.md [hunk 2]
```diff
@@ -41,10 +41,11 @@ module "test_scheduled_task" {
...
 | task_cpu                      | number                          | CPU for the task definition                                                                                                                                                                                                                           | 256                          |
 | task_memory                   | number                          | Memory for the task definition                                                                                                                                                                                                                        | 512                          |
+| cpu_architecture              | string                          | CPU architecture for the task definition. See docs for options                                                                                                                                                                                        | "X86_64"                     |
```
Comment (GaryGSC) [inline README.md]:
```suggestion
| cpu_architecture              | string                          | CPU architecture for the task definition. See [docs](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task_definition_parameters.html#runtime-platform) for options                                                                                                                                                                                        | "X86_64"                     |
```

**Review by GaryGSC — CHANGES_REQUESTED** [review]
It looks like all the changes required to enable running on `ARM64` have been made.

I'm mainly comparing this to [the corresponding change to our Fargate APIs](https://github.com/byu-oit/terraform-aws-fargate-api/pull/38/files).

Over there, since we knew we were going to cut a new major version release, we required a newer version of Terraform and then also made several of our variables optional (thanks to the new capability for optional object attributes). We don't have to do that as part of this release, but we could.

This LGTM, if we consider it a major version bump.

**Review by joshgubler — APPROVED** [review]

**Review by GaryGSC — APPROVED** [review]

**Pull request merged**